<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteManagerDlg</class>
 <widget class="QDialog" name="NoteManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/icons/trayicon</normaloff>:/icons/trayicon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QtNote::NoteManagerView" name="notesTree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::DropAction::CopyAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;h1&gt;Notes Manager&lt;/h1&gt;&lt;h6&gt;(Group, Drag'n'Drop and search your notes)&lt;/h6&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="pixmap">
        <pixmap resource="main.qrc">:/icons/find</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFilter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ckSearchInText">
     <property name="text">
      <string>Search in text</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtNote::NoteManagerView</class>
   <extends>QTreeView</extends>
   <header>notemanagerview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>NoteManagerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
